res/locales/fr_FR.po

```diff
@@ -463,37 +438,32 @@ msgstr ""
 "    – Plan de travail, et deux points ou un segment de ligne (symétrique au "
 "plan de travail)\n"
 
-#: constraint.cpp:545
+#: constraint.cpp:560
 msgid ""
 "A workplane must be active when constraining symmetric without an explicit "
 "symmetry plane."
 msgstr ""
 "Un plan de travail doit être actif lors d'une contrainte de symétrie sans "
 "plan de symétrie explicite."
 
-#: constraint.cpp:579
+#: constraint.cpp:600
 msgid ""
 "Activate a workplane (with Sketch -> In Workplane) before applying a "
 "horizontal or vertical constraint."
 msgstr ""
 "Activez un plan de travail (avec Dessin -> Dans le plan de travail) avant "
 "d'appliquer une contrainte horizontale ou verticale."
 
-#: constraint.cpp:592
+#: constraint.cpp:616
 msgid ""
 "Bad selection for horizontal / vertical constraint. This constraint can "
 "apply to:\n"
 "\n"
-"    * two points\n"
-"    * a line segment\n"
+"    * two or more points\n"
+"    * one or more line segments\n"
 msgstr ""
-"Mauvaise sélection pour la contrainte horizontale / verticale. Cette "
-"contrainte peut s'appliquer à :\n"
-"\n"
-"    – deux points\n"
-"    – Un segment de ligne\n"
```

Comment:
`au moins deux points` and either:

* `au moins deux segments de ligne` (if they are always segments of the same line),
* `au moins deux segments de lignes` (if they have to be segments of different lines), or
* `au moins deux segments de ligne(s)` (if they can to be segments of different lines but don't have to be)

Reply:
Used:
`Au moins un segment de ligne(s)`